eng/scripts/CodeCheck.ps1

```diff
@@ -156,6 +156,11 @@ try {
         & $PSScriptRoot\GenerateProjectList.ps1 -ci:$ci
     }
 
+    Write-Host "Re-generating Web.JS files"
+    Invoke-Block {
+        & dotnet build "$repoRoot\src\Components\Web.JS\Microsoft.AspNetCore.Components.Web.JS.npmproj"
+    }
+
```

Comment:
Hoping we don't need this (see comment above in the ci.yml file)

Reply:
What if people run CodeCheck locally?

Reply:
How is that different from when people run `build` locally?

Reply:
This is going to depend on the discussion with how GenerateReferenceAssemblies.ps1 should work. I think I'm going with the dummy target approach so this won't be needed.